common.rb

```diff
@@ -89,8 +97,9 @@ def put_command(cmd, redact=nil)
     STDERR.puts command_to_echo
   end
 
-  def capture_stdout(cmd)
-    output, _ = Open3.capture2(*cmd, :err=>"/dev/null")
+  # Pass "/dev/null" for err to suppress stderr.
+  def capture_stdout(cmd, err = STDERR)
```

Comment:
What about:
```ruby
def capture_stdout(cmd, err = STDERR)
  if err.nil?
    err = "/dev/null"
  end
  output, _ = Open3.capture2(*cmd, :err => err)
  output
end
```
That way, you don't have to know to pass in the magic string "/dev/null" to suppress.